factory.go

```diff
@@ -58,18 +62,25 @@ func NewFilecoinParser(lib *rosettaFilecoinLib.RosettaConstructionFilecoin, cach
 	}
 
 	helper := helper2.NewHelper(lib, actorsCache, cacheSource.Node, logger)
-	parserV1 := v1.NewParser(helper, logger)
-	parserV2 := v2.NewParser(helper, logger)
+
+	parserV1ActorV1 := v1.NewParser(helper, logger)
+	parserV1ActorV2 := v1.NewActorsV2Parser(helper, logger)
+
+	parserV2ActorV1 := v2.NewParser(helper, logger)
+	parserV2ActorV2 := v2.NewActorsV2Parser(helper, logger)
 
 	return &FilecoinParser{
-		parserV1: parserV1,
-		parserV2: parserV2,
-		Helper:   helper,
-		logger:   logger,
+		parserV1ActorV1: parserV1ActorV1,
+		parserV1ActorV2: parserV1ActorV2,
+
+		parserV2ActorV1: parserV2ActorV1,
+		parserV2ActorV2: parserV2ActorV2,
```

Comment:
This seems a bit confusing actually, parserV1, ActorV1? sounds confusing, again... maybe some comments?